portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path

```diff
@@ -373,7 +373,7 @@
 </tr>
 <tr>
     <td>SIDEBAR_MENUITEM_DUPLICATE_FIELD</td>
-    <td>//a[@title='Duplicate Field'] | //button[text()='Duplicate Field']</td>
+    <td>//a[@title='Duplicate Field'] | //button[text()='Duplicate Field'] | //button[(@class='dropdown-item') and contains(.,'Duplicate')] </td>
```

Comment:
@gleicelisbino, if we change the `Duplicate Field` text to `Duplicate` isn't enough to fix the test?

If not, do we need to keep the previous path (`//a[@title='Duplicate Field'] | //button[text()='Duplicate Field']`) too?

Lastly, can you please add the LPS-131759 as the prefix of this commit?